apache-mesos-mixin/dashboards/apache-mesos-overview.libsonnet

```diff
@@ -1126,13 +1126,13 @@ local memoryUtilizationPanel = {
       showLegend: true,
     },
     tooltip: {
-      mode: 'single',
+      mode: 'multi',
       sort: 'none',
     },
   },
 };
 
-local diskUtilizationPanel = {
+local agentDiskUtilizationPanel = {
```

Comment:
Same thing here with updating the title on the panel itself, rather than the variable name